src/Type/ContentItemTypeType.php

```diff
@@ -116,7 +109,11 @@ public function buildView(FormView $view, FormInterface $form, array $options)
                 }
                 if ($isPersistedEntity && !is_null($subject) && $typeAdmin = $this->sonata->getAdminByClass(get_class($subject))) {
                     $view->vars['type'] = Str::humanize($subject->getType());
-                    $childAdmin = $this->sonata->getAdminByAdminCode($parentAdmin->getCode() . '|' . $typeAdmin->getCode());
+                    $childAdminCode = $parentAdmin->getCode() . '|' . $typeAdmin->getCode();
+                    $childAdmin = $this->sonata->getAdminByAdminCode($childAdminCode);
+                    if (!$childAdmin) {
+                        throw new \InvalidArgumentException(sprintf('Could not find admin with admin code "%s" for "%s"', $childAdminCode, get_class($subject)));
+                    }
                     $childAdmin->setRequest($genericAdmin->getRequest());
```

Comment:
Deze foutmelding ken ik, die zegt iets als `cannot use method setRequest on boolean`. Meestal gebeurt dit als je je cache nog niet gecleared hebt.

Reply:
Klopt, maar ook als je ergens gewoon een foutje hebt. In veel gevallen zou je deze fout kunnen krijgen (op andere plekken), dus ik herken em inmiddels ook, maar op deze manier is het wel zo duidelijk 😄